crates/topos-sequencer-subnet-runtime/tests/subnet_contract.rs

```diff
@@ -68,85 +61,40 @@ const CERTIFICATE_ID_3: CertificateId = CERTIFICATE_ID_8;
 const DEFAULT_GAS: u64 = 5_000_000;
 
 fn spawn_subnet_node(
-    stop_subnet_receiver: tokio::sync::oneshot::Receiver<()>,
-    subnet_ready_sender: tokio::sync::oneshot::Sender<()>,
     port: u32,
     block_time: u64, // Block time in seconds
-) -> Result<tokio::task::JoinHandle<()>, Box<dyn std::error::Error>> {
-    let subnet_test_type = STANDALONE_SUBNET;
-    let subnet_startup_delay = SUBNET_STARTUP_DELAY * (block_time / 2);
-    let block_time_str = block_time.to_string() + "s";
-    let handle = tokio::task::spawn_blocking(move || {
-        let source = Source::DockerHub;
-        let img = Image::with_repository(POLYGON_EDGE_CONTAINER)
-            .source(Source::Local)
-            .tag(POLYGON_EDGE_CONTAINER_TAG)
-            .pull_policy(PullPolicy::IfNotPresent);
-        let mut polygon_edge_node = Composition::with_image(img);
-        polygon_edge_node.env("BLOCK_TIME", block_time_str.clone());
-
-        // Define docker options
-        polygon_edge_node.port_map(SUBNET_RPC_PORT, port);
-
-        let mut polygon_edge_node_docker = DockerTest::new().with_default_source(source);
-
-        // Setup command for polygon edge binary
-        let cmd: Vec<String> = vec![subnet_test_type.to_string()];
-        polygon_edge_node_docker.add_composition(
-            polygon_edge_node
-                .with_log_options(Some(LogOptions {
-                    action: LogAction::Forward,
-                    policy: LogPolicy::OnError,
-                    source: LogSource::StdErr,
-                }))
-                .with_cmd(cmd),
-        );
-        polygon_edge_node_docker.run(|ops| async move {
-            let container = ops.handle(POLYGON_EDGE_CONTAINER);
-            info!(
-                "Running container with id: {} name: {} ...",
-                container.id(),
-                container.name(),
-            );
-            // TODO: use polling of network block number or some other means to learn when subnet node has started
-            tokio::time::sleep(tokio::time::Duration::from_secs(subnet_startup_delay)).await;
-            subnet_ready_sender
-                .send(())
-                .expect("subnet ready channel available");
-
-            info!("Waiting for signal to close...");
-            stop_subnet_receiver.await.unwrap();
-            info!("Container id={} execution finished", container.id());
-        })
-    });
-
-    Ok(handle)
+) -> std::io::Result<Child> {
+    // Ignore output, too verbose
+    let child = Command::new("anvil")
+        .args([
+            "--block-time",
+            &block_time.to_string(),
+            "--port",
+            &port.to_string(),
+        ])
+        .stdout(std::process::Stdio::null())
+        .spawn();
```

Comment:
There is no boot time?

Reply:
I wait for the boot time in the `context_running_subnet_node`